deploy/multiple_keys.sql

```diff
@@ -0,0 +1,83 @@
+
+-- Design note
+-- Users and clients are separate tables, but a single keys table makes more sense. Unfortunately we can't do
+-- foreign key constraints against two tables. Instead we create the effect of 'ON DELETE CASCADE' via a trigger.
+-- However that doesn't prohibit orphaned keys in the table, as we can insert keys without having a matching user or client.
+--
+-- As an alternate implmentation, could split keys into two tables, client_keys, and user_keys
+-- We'd want to add a FK constraint to the id field 'REFERENCES clients(id) ON DELETE CASCADE'
+--
+
+BEGIN;
+
+CREATE TABLE keys(
+   id char(32) NOT NULL,
+   key_id text NOT NULL,
+   public_key text NOT NULL,
+   key_version integer NOT NULL,
+   created_at TIMESTAMP WITHOUT TIME ZONE NOT NULL,
```

Comment:
So based on James' linked article, it sounds like if with went with timestamptz we'd also want to make sure that we set the default timezone to UTC, if we've not already done so. (The author of that article happens to be the author of sqitch, fyi).

I could also see an argument for leaving as is for consistency with the rest of the chef server schema, if other fields are not storing the time zone.